packages/frontmatter/src/frontmatter/validators.ts

```diff
@@ -53,6 +54,7 @@ export const PROJECT_FRONTMATTER_KEYS = [
   'bibliography',
   'math',
   'exports',
+  'design',
 ].concat(SITE_FRONTMATTER_KEYS);
 export const PAGE_FRONTMATTER_KEYS = [
```

Comment:
If we want to be able to define these on the page and override appropriately, we need to add `design` to `USE_SITE_FALLBACK`